### DISCUSSION TITLE
「驳岁月静好派」自欺欺人的伟光正忽悠！

### DISCUSSION THREAD

**Discussion by frshgrass** [WeNeedHome/SummaryOfLoanSuspension#1028] [General]
搞笑呢 国内这高房价不就是政府收的进城税，结果收钱不办事 还把监管资金给弄没。自己躲在体制内吃香喝辣岁月静好。

还说什么美国有占领华尔街 既然你要求对等 你是不是支持我们也可以。

权益是斗争来的 要是东西被抢了 哼一声也不敢 只会被欺负得更惨淡。

大家用力的骂 高声得骂 要让他们知道 兔子急了也咬人 人民也是有力量的。

**Comment by wedxz** [comment]
> 搞笑呢 国内这高房价不就是政府收的进城税，结果收钱不办事 还把监管资金给弄没。自己躲在体制内吃香喝辣岁月静好。
> 
> 还说什么美国有占领华尔街 既然你要求对等 你是不是支持我们也可以。
> 
> 权益是斗争来的 要是东西被抢了 哼一声也不敢 只会被欺负得更惨淡。
> 
> 大家用力的骂 高声得骂 要让他们知道 兔子急了也咬人 人民也是有力量的。

当你感觉负重前行的时候，肯定是有人骑在你脖子上岁月静好…

**Comment by itsoursduty** [comment]
你不去关心政治，政治就会来关心你。岁月静好，迟早会挨铁拳
